### DISCUSSION TITLE
How to map leaderkey for explorer?

### DISCUSSION THREAD

**Discussion by liuhq** [vscode-neovim/vscode-neovim#2274] [Q&A] [answered]
For example, I map `<leader>e` to `workbench.view.explorer`, and the action is working for moving focus from editor to explorer. But, when I want to return to editor, `<leader>e` is not works. (the native shortcut `<C-S-E>` works)

**Comment by justinmk** [comment, marked as answer]
vscode-neovim is not involved with Explorer, except that it sets some default mappings. You could manually configure your vscode settings.

**Reply by liuhq** [reply]
okay, thanks for your answer
